health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/BaseMapSwitcher.js

```
@@ -0,0 +1,59 @@
+
+
+
+const BaseMapSwitcher = ({ baseMaps, showBaseMapSelector, setShowBaseMapSelector, handleBaseMapToggle, selectedBaseMapName, basemapRef, t }) => {
+  if (!baseMaps) return null;
+  return (
+    <div className="base-map-selector">
+      <div
+        className="icon-first"
+        onClick={() => setShowBaseMapSelector((previous) => !previous)}
+        onKeyUp={() => setShowBaseMapSelector((previous) => !previous)}
+        tabIndex={0}
```

Biome (1.9.4)

[error] The HTML element div is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)